### ISSUE TITLE
ONIX for accessModeSufficient

### ISSUE THREAD

**Issue by madeleinerothberg** [w3c/a11y-discov-vocab#139]
This passage is tricky. Can Chris Saynor suggest revised text? At the least, there is a typo in spelling accessible near the end, and it includes "screen reader friendly" which we no longer use, so that needs to be rewritten:

[List: 81; Code: 10](https://ns.editeur.org/onix/en/81/10) combined with [List: 196; Code: 10](https://ns.editeur.org/onix/en/196/10) means all text is actual text. Note that List: 81; Code: 10 on its own (without List: 196; Code: 10) admits the possibility that the "text" is inaccessible because it is an image of text.
Note : on this point ONIX is unclear. [List: 196; Code: 09](https://ns.editeur.org/onix/en/196/09) Inaccessible Known to lack significant features required for broad accessibility. Would give a better indication for "Screen reader friendly" (if code 09 is not found so the publication is textual). In both cases it's hard for a publisher to figure out how to describe "all content is accesible thru true text", he must adopt a reasoning by absence.

**Comment by clapierre** [comment]
@chrisONIX thoughts?

**Comment by chrisONIX** [comment]
@clapierre @madeleinerothberg 
For this one - would it be simpler to map it simply to list 81 - code 10 and not combine it with another code ?
List 196 - code 10 is meant to say that the creator of the ePub has done nothing to block any accessibility features, so is a general affirmation and may not be the right combination here.

The other option is to use list 81 - code 10 in combination with list 196 - code 52 - https://ns.editeur.org/onix/en/196/52 
(have a look at the note) which would confirm that the textual content is fully accessible.

Which option do you think would make most sense in relation to accesModeSufficent?

**Comment by chrisONIX** [comment]
Suggest a review of the mapping from schema to ONIX values for accessModeSufficient

**Comment by chrisONIX** [comment]
@madeleinerothberg @clapierre 

to help with getting this mapping right is accessModeSufficient a confirmation of all the different accessModes in the ePub or is it the accessModes plus any additions for accessibility purposes that effectively allow for other accessModes that are represented by accessModeSufficent?

**Comment by chrisONIX** [comment]
@madeleinerothberg for discussion - would the mapping for ONIX make sense if we used code 52 from list 196 ?https://ns.editeur.org/onix/en/196/52  The way the note is written for this code is that everything has been done to make this product fully accessible for all sensory ways of reading. 

Would code 52 work for the accessModeSufficent textual, tactile and auditory?

Because it says reading without site it will not work for visual - if I have understood accessModeSufficent visual correctly?


Does the mapping itself need to mention the accessModes and list 81 or can the ONIX column refer back to access mode? 

We could do a comment on each of the four types like your text above?

**Comment by chrisONIX** [comment]
A suggested text to go above this section:

The Schema to ONIX crosswalk are for instances where the accessModeSufficient includes this vocabulary.
The mapping should also use the accessMode to ONIX values above.
The ONIX to Schema mapping is more complex, and is based on having the codes from list 81 (above) to confirm the accessMode and the accessibility features from list 196 (see above) to confirm that they are accessible, as  accessModeSufficient reflects the combination of the main accessModes with any available accessibilityFeatures

**Comment by clapierre** [comment]
AMS = Auditory maps to ONIX 196/51 ONLY, 81/01 ONLY or both.
AMS = Textual maps to ONIX 196/52 ONLY (remove all others)
AMS = Visual does not map to ONIX
AMS = Tactile does not map to ONIX

Textual, Visual = Indicates that the resource contains some information encoded in textual form, but that visual perception is necessary to consume other parts of the resource.

**Comment by chrisONIX** [comment]
@clapierre we can not do a simple mapping to ONIX for AMS = Textual, visual, so we will do some of our own documentation as with AMS = visual 
So once you are done this can be closed.
